app/assets/javascripts/discourse/models/post.js.es6

```diff
@@ -320,6 +320,13 @@ Post.reopenClass({
     });
   },
 
+  mergePosts(selectedPosts) {
+    return Discourse.ajax("/posts/merge_posts", {
+      type: 'PUT',
+      data: {post_ids: selectedPosts.map(p => p.get('id'))}
+    });
```

Comment:
Sure :) For a start, https://github.com/discourse/discourse/blob/be3a5a56ccc284b352aa65080fc6b955f73cc72a/app/assets/javascripts/admin/controllers/admin-site-text-edit.js.es6#L13 will be fine